editoast/src/views/mod.rs

```diff
@@ -156,6 +164,43 @@ struct ListId {
     ids: HashSet<i64>,
 }
 
+#[derive(Debug, Default, Clone, Serialize, Deserialize, IntoParams, ToSchema)]
+#[into_params(parameter_in = Query)]
+pub struct InfraIdQueryParam {
+    infra_id: i64,
+}
+
+#[derive(Debug, Serialize, ToSchema)]
+#[serde(tag = "status", rename_all = "snake_case")]
+enum SimulationSummaryResult {
+    /// Minimal information on a simulation's result
```

Comment:
Feels weird to have such specific structures in `views/mod.rs`, can't these be put in another more specific module?